content/api/vision.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Vision
+---
+
+The main way your robot will see markers is through the webcam.
+
+{{% notice info %}}
+Throughout this page it is assumed you have a `Robot` object defined as `r`
+{{% /notice %}}
+
+## Using the camera to search for markers
+If you have a webcam connected, you can use `r.camera.see()` to take a picture. 
+
+If you consider it necessary you can assign the camera object to a variable just like `Robot`:
+
+```python
+camera = r.camera
+```
+The software will process the image and return any markers it sees into an array sorted by distance.
+
+```python
+markers = r.camera.see()
+```
+
+You can then iterate over the `markers` array just as you would any other list:
+```python
+for marker in markers:
+  # Do robot stuff
+```
+
+## Marker properties
+The markers that are contained within the array have some useful properties:
+
+  - `is_wall_marker()` - returns whether or not the marker is a wall marker.
+  - `is_token_marker()` - returns whether or not the marker is a token marker.
+  - `id()` - returns the id of the marker
```

Comment:
This is a property, it's not called as a function. As are `pixel_center`, `polar` and `cartesian`, which is also spelt wrong

Reply:
Would be good to explain what a "marker ID" is anyway